packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -423,6 +425,8 @@ class SliderThemeData with Diagnosticable {
   /// The color given to the [valueIndicatorShape] to draw itself with.
   final Color? valueIndicatorColor;
 
+  /// The color given to the [valueIndicatorShape] stroke.
+  final Color? valueIndicatorStrokeColor;
```

Comment:
Looking at the related property above, `valueIndicatorColor`, that applies to the `RectangularSliverValueIndicatorShape`, but also:
- `PaddleSliderValueIndicatorShape`
- `PaddleRangeSliverValueIndicatorShape`
- `DropSliderValueIndicatorShape`

Should these other classes also use this new property? I think they should, but if not for some reason, we should probably document the difference in how the new property is applied as folks might expect it to be the same as the other property.

Reply:
HI @Piinks

Thanks for your feedback 🙂. 

I completely agree with you! It makes a lot of sense to maintain consistency across these related classes.

I'll make sure to explore the possibility of extending the use of the "valueIndicatorStorkeColor" property to the other classes you mentioned.